### PR TITLE
New keymap for ID75 - paryz

### PR DESCRIPTION
**Pull request by FilipParyz**
## Description
It's just a new, personal keymap for ID75 with layer indication and customized RGB matrix.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/idobao/id75/keymaps/paryz/config.h
Comment (waffle87) [inline keyboards/idobao/id75/keymaps/paryz/config.h:20]:
```suggestion
```

File: keyboards/idobao/id75/keymaps/paryz/keymap.c
Comment (waffle87) [inline keyboards/idobao/id75/keymaps/paryz/keymap.c:51]:
```suggestion
enum layers {
    _QWERTY,
    _FN,
    _LOWER,
    _RAISE
};
```

File: keyboards/idobao/id75/keymaps/paryz/keymap.c
Comment (waffle87) [inline keyboards/idobao/id75/keymaps/paryz/keymap.c:82]:
```suggestion
    QK_GESC,            KC_1,   KC_2,    KC_3,    KC_4,   KC_5,  KC_MINS, KC_GRV,  KC_EQL,  KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_DEL,
    KC_TAB,             KC_Q,   KC_W,    KC_E,    KC_R,   KC_T,  KC_LBRC, KC_BSLS, KC_RBRC, KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_ENT,
    LT(_LOWER,KC_CAPS), KC_A,   KC_S,    KC_D,    KC_F,   KC_G,  KC_HOME, KC_DEL,  KC_PGUP, KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, LT(_RAISE, KC_QUOT),
    KC_LSFT,            KC_Z,   KC_X,    KC_C,    KC_V,   KC_B,  KC_END,  KC_UP,   KC_PGDN, KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_UP,   RSFT_T(KC_SLSH),
    KC_LCTL,            MO(_FN),KC_LGUI, KC_LALT, KC_SPC, KC_SPC,KC_LEFT, KC_DOWN, KC_RGHT, KC_BSPC, KC_BSPC, KC_RALT, KC_LEFT, KC_DOWN, KC_RGHT
 ),

 [_FN] = LAYOUT_ortho_5x15(
    KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,    KC_NLCK, KC_SLSH, KC_ASTR, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
    KC_PSCR, _______, _______, _______, _______, _______,  KC_P7,   KC_P8,   KC_P9,   KC_MINS, _______, _______, KC_PSCR, KC_SLCK, KC_PAUS,
    KC_MPRV, KC_MPLY, KC_MNXT, RGB_HUI, _______, _______,  KC_P4,   KC_P5,   KC_P6,   KC_PLUS, _______, QK_BOOT, _______, _______, _______,
    KC_VOLD, KC_MUTE, KC_VOLU, RGB_SAI, RGB_SPI, _______,  KC_P1,   KC_P2,   KC_P3,   KC_PENT, _______, _______, _______, _______, _______,
    _______, _______, RGB_TOG, RGB_VAI, RGB_MOD, _______,  KC_P0,   _______, KC_PDOT, KC_PENT, KC_PENT, _______, _______, RGB_M_R, RGB_M_SW
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by FilipParyz** [comment]
@waffle87 Thank you for the review, I'll fix those issues whenever possible :)

File: keyboards/idobao/id75/keymaps/paryz/keymap.c
Comment (drashna) [reply keyboards/idobao/id75/keymaps/paryz/keymap.c:51]: Specifically, we push for enums instead of defines, because they're easier to manage, and less prone to empty layers (as they may not have anything in them but they do take up firmware space)

Comment (FilipParyz) [reply keyboards/idobao/id75/keymaps/paryz/keymap.c:51]: I'm sorry, In the default keymaps define's are used and I've repeated this mistake here :)

File: keyboards/idobao/id75/keymaps/paryz/config.h
Comment (FilipParyz) [reply keyboards/idobao/id75/keymaps/paryz/config.h:20]: Why shouldn't I add the minimum brightness value?

Comment (fauxpark) [reply keyboards/idobao/id75/keymaps/paryz/config.h:20]: It doesn't do anything.

**Comment by FilipParyz** [comment]
I think I should be ready with all those changes. I've updated my installation of QMK_MSYS and fixed the keymap for both version of the keyboard :)

File: keyboards/idobao/id75/keymaps/paryz/keymap.c
Comment (drashna) [inline keyboards/idobao/id75/keymaps/paryz/keymap.c:19]: You know, you could just move all of this to the keyboard_post_init_user function, as: 
```c
    g_led_config = { {
```

**Review by drashna — APPROVED** [review]

File: keyboards/idobao/id75/keymaps/paryz/keymap.c
Comment (FilipParyz) [reply keyboards/idobao/id75/keymaps/paryz/keymap.c:19]: Well, yes I guess I could. I believe that's what the compiler will do anyways, to optimize it.

Comment (tzarc) [reply keyboards/idobao/id75/keymaps/paryz/keymap.c:19]: Extremely unlikely -- it's only going to have side effects after `keyboard_post_init_user()`. Prior to that, the keyboard-supplied values will be present.

I'll wait to see if you want to change things before merge -- your call.

Comment (FilipParyz) [reply keyboards/idobao/id75/keymaps/paryz/keymap.c]: Yup, thank you for waiting, changed it. None of the changes made are breaking, so I think we can go ahead :)

**Review by tzarc — APPROVED** [review]

**Pull request merged**
